modules/clients.py

```diff
@@ -27,50 +27,41 @@ def done_at(self, time):
 
 class Clients(object):
 
-    def __init__(self, env, server, requests, process_time=0.00001, create_request_rate=0.01, max_requests=float("inf")):
+    def __init__(self, env, server, requests, process_time=0.00001, create_request_rate=0.01, max_requests=float("inf"), service_time=0.0035, memory=0.02):
```

Comment:
Could you please document what process_time mean?

Reply:
It isn't clear in did. The process_time should represent the delay after each request sending. Some variables along the code have no appropriate name. We will fix that.

Reply:
I actually didn't understand exactly what it means.

>  The process_time should represent the delay after each request sending. 

Is this the time the server should be spending on processing the request? If yes, this is exactly the `service_time` we have been talking.

Reply:
@danielfireman, we thought in put a delay after each request send. There is a queue of requests to be sended. After take one and send it for the server, we wait a very short time to send another request. 
For example let's use 0.1 units of time to clarify. 
At moment 0.0, we send one request to the server. At moment 0.1 we send another. At 0.2 one more and so on.. 
That variable (now named as sleep) puts periodicity in the sending of requests.